packages/rum/test/e2e/integration-test.js

```diff
@@ -28,31 +28,30 @@ const puppeteer = require('puppeteer')
 async function runIntegrationTest(pageUrl) {
   const browser = await puppeteer.launch({
     headless: true,
-    args: [
-      '--no-sandbox',
-      '--disable-setuid-sandbox',
-      '--remote-debugging-port=9222'
```

Comment:
Don't we need the remote debugging port?

Reply:
I don't think so. With the option enabled the tests hang in local so i removed it in favor of enabling test logs.